execution/executor-benchmark/src/lib.rs

```diff
@@ -520,8 +520,10 @@ pub enum SingleRunMode {
     },
 }
 
-// Optional more detailed configuration.
+/// Optional more detailed configuration.
 pub struct SingleRunAdditionalConfigs {
+    /// If num_generator_workers=1 then order in which transactions are generated
+    /// is kept in the block, otherwise they will be randomized
```

Comment:
is it randomized across workers?

Reply:
Yeah, each thread generates on its own,  and then stitches them together.

Maybe we can have some smarter stitching there at some point,  but this is good for now